charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -132,7 +132,7 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: trino-access-control-volume-coordinator
+  name: {{ template "trino.coordinator" . }}-access-control-volume-coordinator
```

Comment:
could we eliminate suffix `-coordinator`?
since in the template already has it, similar to other confimap objects

Reply:
> since in the template already has it, similar to other confimap objects

I thought of it. The problem with it would be that in case someone doesn't overrides the name the default name only say "trino" which will cause duplicate configmap issues.